File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/9.go
package main
import "fmt"
import "strings"

func contem101(s string) bool {
    return strings.Contains(s, "101")
}

func main() {
    fmt.Println(contem101("1101"))  // True
    fmt.Println(contem101("1110"))  // False
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/9.java
public class AFDContem101 {
    public static boolean contem101(String s) {
        return s.contains("101");
    }

    public static void main(String[] args) {
        System.out.println(contem101("1101"));  // True
        System.out.println(contem101("1110"));  // False
    }
}
